### PR TITLE
feat: Re-arrange sidebar and add navigation to locksmith API

### PR DESCRIPTION
**Pull request by searchableguy**
CleanShot 2022-09-28 at 18 32 27@2x

### CODE REVIEW

**Comment by searchableguy** [comment]
FYI: Inserting any custom item in the sidebar at a specific position require manually writing that portion hence the changes here. 

The other option is writing sidebar generator code which would have been less code but given we want control over all elements, I wrote down how the sidebar should be positioned manually.

**Comment by julien51** [comment]
ooooooooooohhhhh

**Review by julien51 — CHANGES_REQUESTED** [review]
I see the tools have been reorganized. Let's use `sidebar_position` to maintain the right order please

File: sidebars.js [hunk 1]
```diff
@@ -14,18 +14,138 @@
...
+          items: [
+            {
+              type: "link",
+              href: "/api/locksmith",
+              label: "API",
+            },
+            "tools/locksmith/authorization-strategy",
+            "tools/locksmith/metadata",
+            "tools/locksmith/webhooks",
+          ],
```
Comment (searchableguy) [inline sidebars.js:130]: @julien51 They are actually reorganized from here now instead of sidebar position which is confusing to use.

**Comment by searchableguy** [comment]
@julien51 I actually opened an issue asking for that if we could generate our own sidebar there. 

I'm waiting for response and will check if I can make the necessary changes easily to add it. 


CleanShot 2022-09-28 at 18 48 01@2x

For now, a user who is on API page will click home to go back.

**Comment by searchableguy** [comment]
see #167

**Comment by julien51** [comment]
LGTM!

**Review by julien51 — APPROVED** [review]

**Pull request merged**